package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gaia-button",
+  "version": "0.0.5",
+  "homepage": "https://github.com/gaia-components/gaia-button",
+  "license": "MIT",
+  "main": "gaia-button.js",
+  "devDependencies": {
```

Comment:
I think `gaia-button` is not ready for npm since `CSS variable` is not support now...